Index: mkusb9/mkusb.desktop
===================================================================
--- mkusb9.orig/mkusb.desktop	2016-07-20 20:42:36.974967551 -0500
+++ mkusb9/mkusb.desktop	2016-07-20 20:42:36.972967510 -0500
@@ -4,7 +4,7 @@
 Type=Application
 Name=mkusb
 Comment=Install system to mass storage device, typically USB pendrive
-Exec=bash -c 'TERM=xterm;/usr/share/mkusb/mkusb-start'
+Exec=/usr/share/mkusb/mkusb-start
 Icon=mkusb
 Terminal=false
 StartupNotify=false
